app/assets/stylesheets/_milestone.scss

```diff
@@ -16,7 +16,6 @@
     position: relative;
 
     .ui-icon {
-      float:left;
```

Comment:
tinkering I noticed it didn't need to be floated.

For some reason the float was messing with the header heights somehow